charts/kubewarden-crds/templates/policyserver.yaml

```diff
@@ -3,9 +3,20 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
+    cert-manager.io/inject-ca-from: $(CERTIFICATE_NAMESPACE)/$(CERTIFICATE_NAME)
```

Comment:
What is the purpose of this annotation? As far as I see, this annotation is added to mutating and validating webhook configuration only

Reply:
yes, I don't think we need this annotation here in the CRD. It should be enough to add it to the [webhook configuration](https://github.com/kubewarden/helm-charts/blob/main/charts/kubewarden-controller/templates/webhooks.yaml#L6) as @ereslibre pointed out.

Reply:
That seems right.
I've disabled the generation of those lines in kubewarden-controller with https://github.com/kubewarden/kubewarden-controller/pull/195 (and found a way to possibly automate the CRD manifest consumption in helm charts).
Here, I have rebased and dropped those annotations from the 3 CRDs, instead of waiting for a new controller release (as we now have the refactor merged).